op-bindings/bindgen/fixtures_test.go

```diff
@@ -33,15 +146,267 @@ var removeDeploymentSaltTestsFailures = []struct {
+			DeployedBin: "",
+		},
+		compareInitialization: true,
+		compareDeployment:     false,
```

Comment:
Is this false because the initcode relies on immutables so deploy code won't match, therefore we skip the deployed code assertion? If so, instead of skipping this check it would be stronger to assert on the mismatch, i.e. rename the vars to something like `initCodeShouldMatch` and `deployedCodeShouldMatch` and then assert based on the value

Reply:
For the specific test case, no that's not the reason, it's `false` just to hit the code path. However for the actual implementation, you're correct, we don't want to compare deployed bytecode for contracts with `immutable`s - I can add the test case you're suggesting